packages/api/amplify_api_dart/test/graphql_helpers_test.dart

```diff
@@ -774,7 +774,7 @@ void main() {
     group('ModelSubScriptions', () {
       test('ModelSubscriptions.onCreate() should build a valid request', () {
```

Comment:
request: For the tests here with no `where` clause provided, could we add assertions for how the `req.variables` should look? If we want the provided value to be null, empty, `{}`  we should enforce that here with a test.